### PR TITLE
Add jobs aa1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no job page.
- Currently there is no team page.

**What is the new behavior (if this is a feature change)?**
- A job page has been added.
- A team page has been added.

**Other information**:
- The version has been updated to `v1.1.0`.

### CODE REVIEW

**Review by sweggerat — APPROVED** [review]
good

File: src/components/pages/TeamPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+//#region > Data
+const TEAM = [
+  {
+    name: "Christian Aichner",
+    avatar:
+      "https://avatars3.githubusercontent.com/u/21159914?s=460&u=afab4659183999f1adc85089bb713aefbf085b94&v=4",
+    facebook: "https://www.facebook.com/aichner.christian",
+    github: "https://github.com/Aichnerc",
+    snek: "https://snek.at/u/aichnerc",
```
Comment (pinterid) [inline src/components/pages/TeamPage/index.jsx]:
```suggestion
    snek: "https://snek.at/u/Aichnerc",
```

**Review by pinterid — CHANGES_REQUESTED** [review]
Found one issue.

File: src/components/pages/PositionsPage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+          </MDBCol>
+          <MDBCol lg="4" className="mt-3">
+            <MDBCard className="border">
+              <MDBCardBody>
+                <p className="font-weight-bold mb-0">Researcher</p>
+                <p className="text-muted small">Voluntary</p>
+                <div className="text-left">
+                  <p className="mb-2">Our main areas of research</p>
+                  <ul>
+                    <li>Avionik</li>
```
Comment (Theralun) [inline src/components/pages/PositionsPage/index.jsx]: Avionics

**Review by Theralun — CHANGES_REQUESTED** [review]
Have a look.

File: src/components/pages/PositionsPage/index.jsx (hunk 2 above)
Comment (Aichnerc) [reply src/components/pages/PositionsPage/index.jsx]: Oh damn.

**Review by Theralun — APPROVED** [review]
LGTM.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by pinterid — APPROVED** [review]

**Pull request merged**